erpnext/setup/setup_wizard/setup_wizard.py

```diff
@@ -88,10 +88,23 @@ def create_fiscal_year_and_company(args):
 			'chart_of_accounts': args.get(('chart_of_accounts')),
 			'domain': args.get('domain')
 		}).insert()
-
+		
+		#Enable shopping cart
+		enable_shopping_cart(args)
+		
 		# Bank Account
 		create_bank_account(args)
-
+		
+def enable_shopping_cart(args):
+	frappe.get_doc({
+		"doctype":"Shopping Cart Settings",
+		"enabled": 1,
+		'company':args.get('company_name').strip(),
+		'price_list':frappe.db.get_value("Price List", {"selling": 1}),
+		'default_customer_group':"Individual",
```

Comment:
names are translated, should be `_("Individual")`

Also always leave a space after : in dict example  `"doctype": "Shopping Cart Settings",`